data/fields/parking/orientation.json

```diff
@@ -0,0 +1,12 @@
+{
+    "key": "parking:orientation",
+    "type": "combo",
+    "label": "Parking bay orientation",
```

Comment:
I'd recommend simplifying to just: 'Orientation' (its perhaps self-evident that it is indeed 'parking' orientation)

Reply:
Done.